src/examples/counter/Counter.ts

```diff
@@ -0,0 +1,3 @@
+export interface Counter {
+  count: number
+}
```

Comment:
What do you think about making this a simple numeric type instead? I see it as a good opportunity to demonstrate that models can be _anything_, not just objects with properties.

```suggestion
type Counter = number
```

You could simplify further by removing the `Counter` type completely and just using `number` directly.